packages/components/accordion/src/accordion-item.tsx

```diff
@@ -51,48 +92,53 @@ export interface AccordionItemProps
     AccordionItemOptions {}
 
 export const AccordionItem = forwardRef<AccordionItemProps, "div">(
-  ({ className, children, icon, isDisabled = false, label, ...rest }, ref) => {
+  (
+    { className, children, disabled, icon, isDisabled = false, label, ...rest },
+    ref,
+  ) => {
     const itemId = useId()
     const panelId = useId()
 
-    const { index, isMultiple, isToggle, setFocusedIndex, setIndex, styles } =
+    disabled ??= isDisabled
+
+    const { index, multiple, setFocusedIndex, setIndex, styles, toggle } =
       useAccordionContext()
 
     const {
       descendants,
       index: i,
       register,
-    } = useAccordionDescendant({ disabled: isDisabled })
+    } = useAccordionDescendant({ disabled })
 
-    const isOpen =
+    const open =
       i !== -1 ? (isArray(index) ? index.includes(i) : index === i) : false
 
-    if (isOpen && isDisabled)
+    if (open && disabled)
       console.warn(`Accordion: Cannot open a disabled accordion item`)
 
     const onChange = useCallback(
-      (isOpen: boolean) => {
+      (open: boolean) => {
         if (i === -1) return
 
-        if (isMultiple && isArray(index)) {
-          setIndex(isOpen ? index.concat(i) : index.filter((v) => v !== i))
-        } else if (isOpen) {
+        if (multiple && isArray(index)) {
+          setIndex(open ? index.concat(i) : index.filter((v) => v !== i))
+        } else if (open) {
           setIndex(i)
-        } else if (isToggle) {
+        } else if (toggle) {
           setIndex(-1)
         }
       },
-      [isMultiple, isToggle, i, index, setIndex],
+      [multiple, toggle, i, index, setIndex],
     )
```

Comment:
Do I need to change this isOpen to open?

Reply:
I have a question. I’m also changing the variable name to 'open'. Is this unnecessary? @hirotomoyamada 

![스크린샷 2024-11-17 14 55 58](https://github.com/user-attachments/assets/d195f408-f6d7-4dd6-a25f-1d1477e639b2)

Reply:
I remembered why I changed the variable name to `open`. I thought it would bring more consistency to the variable names, so I made the change. If it's unnecessary, I can revert it back.